### PR TITLE
Throw a TypeError when falsy value found in plugins

### PR DESCRIPTION
**Pull request by BlackGlory**

### CODE REVIEW

**Comment by hzoo** [comment]
Ref https://github.com/babel/babel/pull/3273

Ok great! Could you add a test? If not we can do it

**Comment by BlackGlory** [comment]
@hzoo 
I do not even know where the test write is right in this project, so it is still done by your right. I'll thank you for your add the test :+1: .

**Comment by hzoo** [comment]
:+1: We need some better docs for that

**Comment by erikras** [comment]
It would be really great to have included the index, `i` in the error message.

**Comment by loganfsmyth** [comment]
I'm hoping to revamp a lot of these error messages in Babel 7. Even with the index it is super hard to follow them at the moment.
